qt/widgets/plotting/src/Mpl/ExternalPlotter.cpp

```diff
@@ -132,17 +142,35 @@ void ExternalPlotter::plotSpectra(std::string const &workspaceName, std::string
  * @param workspaceIndices List of indices to plot
  */
 void ExternalPlotter::plotCorrespondingSpectra(std::vector<std::string> const &workspaceNames,
-                                               std::vector<int> const &workspaceIndices, bool errorBars) {
+                                               std::vector<int> const &workspaceIndices, bool errorBars,
+                                               boost::optional<std::vector<QHash<QString, QVariant>>> kwargs,
+                                               boost::optional<std::vector<bool>> errorBarsPerSpectra) {
```

Comment:
We now have both an `errorBars` and `errorBarsPerSpectra` parameter which confuses things. We should probably get rid of `errorBars`, and do a small refactor to the Indirect calls of this function.

Reply:
and then I think `errorBarsPerSpectra` should not be `boost::optional`, and renaming it to simply `errorBars` would follow the convention of the other parameter names